WebApplication/Migrations/20201123222800_DbBasexx.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace WebApplication.Migrations
+{
+    public partial class DbBasexx : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "Buildings",
+                columns: table => new
+                {
+                    Id = table.Column<string>(nullable: false),
+                    SerialNumber = table.Column<string>(nullable: true),
+                    Name = table.Column<string>(nullable: true),
+                    Floors = table.Column<string>(nullable: true),
+                    Color = table.Column<string>(nullable: true),
+                    Shape = table.Column<string>(nullable: true)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_Buildings", x => x.Id);
+                });
```

Comment:
1. What is the difference between Id and SerialNumber?
2. Why are Floors strings?

Reply:
Same answer.
Floors are easier to get through their ids.